internal/npm_install/npm_install.bzl

```diff
@@ -79,6 +79,14 @@ fine grained npm dependencies.
         mandatory = True,
         allow_single_file = True,
     ),
+    "package_path": attr.string(
+        default = "",
+        doc = """If set, link the 3rd party node_modules dependencies under the package path specified.
+
+In most cases, this should be the directory of the package.json file so that the linker links the node_modules
+in the same location they are found in the source tree. In a future release, this will default to the package.json
```

Comment:
make a 4.0 tracking issue for this and link it here

Reply:
👍